documentation/book/proc-manual-delete-pod-pvc-zookeeper.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
```

Comment:
_Please use the following content, including reworded title:_

##  Delete Zookeeper nodes manually
This procedure describes how to delete an existing Zookeeper node by using an {ProductPlatformName} annotation. Deleting a Zookeeper node consists of deleting both the Pod on which Zookeeper is running and the related `PersistentVolumeClaim` (if the cluster was deployed with persistent storage). After deletion, the Pod and its related `PersistentVolumeClaim` are recreated automatically.

Warning: Deleting a `PersistentVolumeClaim` can cause permanent data loss. The following procedure should only be performed if you have encountered storage issues.

### Prerequisites

* A running Zookeeper cluster.
* A running Cluster Operator.

### Procedure

1. Find the name of the Pod that you want to delete.

  For example, if the cluster is named cluster-name, the pods are named cluster-name-zookeeper-index, where index starts at zero and ends at the total number of replicas.

2. Annotate the Pod resource in {ProductPlatformName}.

+ On {OpenShiftName} use `oc annotate`:

  ``oc annotate pod cluster-name-zookeeper-index cluster.operator.strimzi.io/delete-pod-and-pvc=true``

3. Wait for the next reconciliation, when the annotated pod with the underlying persistent volume claim will be deleted and then recreated.

### Additional resources
For more information on deploying the Cluster Operator, see [cluster-operator-str].

For more information on deploying the Zookeeper cluster, see ] and xref:deploying-zookeeper-cluster-kubernetes-str[.